api/tests/acceptance/application/authentication-controller_test.js

```diff
@@ -84,13 +84,12 @@ describe('Acceptance | Controller | authentication-controller', () => {
       });
 
       const expectedResponseError = {
-        errors: [
-          {
-            title: 'PasswordShouldChange',
-            status: '401',
-            detail: 'Erreur, vous devez changer votre mot de passe.',
-          },
-        ],
+        errors: [{
+          code: 'SHOULD_CHANGE_PASSWORD',
+          detail: 'Erreur, vous devez changer votre mot de passe.',
+          status: '401',
+          title: 'PasswordShouldChange',
```

Comment:
**chipotage**
Préciser qui fait quoi
```suggestion
          title: 'userShouldChangePassword',
```

Reply:
Je ne vais pas faire de changement dans ce ticket.
Mais si je veux rester au plus proche de l'[ADR (point 5)](https://jsonapi.org/format/#errors), je rechercherais :
* _a short, human-readable summary of the problem that SHOULD NOT change from occurrence to occurrence of the problem, except for purposes of localization_
* Un focus sur  le couple HTTP statusCode et `error.code` pour gérer la traduction des  messages d'erreur (côté front)